### PR TITLE
Add Exception handler to remote debugger

### PR DESCRIPTION
**Pull request by polaroi8d**
In initial state the exception handler is enable. Use the `exception` command with the 0 to disable the feature and 1 to enable it. Modified the `help` command test too. 

JerryScript-DCO-1.0-Signed-off-by: Levente Orban orbanl@inf.u-szeged.hu

### CODE REVIEW

File: jerry-core/debugger/jerry-debugger.h [hunk 1]
```diff
@@ -243,6 +246,12 @@ typedef struct
   jerry_debugger_frame_t frames[JERRY_DEBUGGER_SEND_MAX (jerry_debugger_frame_t)]; /**< frames */
 } jerry_debugger_send_backtrace_t;
 
+typedef struct
+{
+  uint8_t type; /**< type of the message */
+  uint8_t enable; /**< non-zero: enable stop at exception */
```
Comment (LaszloLango) [inline jerry-core/debugger/jerry-debugger.h]: Is this a boolean value? The description is not obvious to me.

File: jerry-core/debugger/jerry-debugger.c [hunk 2]
```diff
@@ -391,6 +391,27 @@ jerry_debugger_process_message (uint8_t *recv_buffer_p, /**< pointer the the rec
...
 
+    case JERRY_DEBUGGER_EXCEPTION_CONFIG:
+    {
+      JERRY_DEBUGGER_CHECK_PACKET_SIZE (jerry_debugger_receive_exception_config_t);
+      JERRY_DEBUGGER_RECEIVE_BUFFER_AS (jerry_debugger_receive_exception_config_t, exception_config_p);
+
+      if (exception_config_p->enable == 0)
+      {
+        JERRY_CONTEXT (debugger_flags) = (uint8_t) (JERRY_CONTEXT (debugger_flags)
+                                         | JERRY_DEBUGGER_VM_IGNORE_EXCEPTION);
```
Comment (zherczeg) [inline jerry-core/debugger/jerry-debugger.c]: I don't think this is the right style. Add a local variable if the line is too long.

File: tests/debugger/do_exception.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+print("exception handler configuration test")
+
+function foo() {
+	try {
```
Comment (LaszloLango) [inline tests/debugger/do_exception.js]: Please don't use tab characters

**Comment by polaroi8d** [comment]
@LaszloLango @zherczeg Updated the PR.

**Pull request merged**